argilla-server/docker/quickstart/README.md

```diff
@@ -4,131 +4,12 @@
   Argilla
   <br>
 </h1>
-<p align="center">
-<a  href="https://pypi.org/project/argilla/">
-<img  alt="CI"  src="https://img.shields.io/pypi/v/argilla.svg?style=flat-square&logo=pypi&logoColor=white">
-</a>
-<!--a  href="https://anaconda.org/conda-forge/rubrix">
-<img  alt="CI"  src="https://img.shields.io/conda/vn/conda-forge/rubrix?logo=anaconda&style=flat&color=orange">
-</!a-->
-<img alt="Codecov" src="https://codecov.io/gh/argilla-io/argilla/branch/main/graph/badge.svg?token=VDVR29VOMG"/>
-<a href="https://pepy.tech/project/argilla">
-<img  alt="CI"  src="https://static.pepy.tech/personalized-badge/argilla?period=month&units=international_system&left_color=grey&right_color=blue&left_text=pypi%20downloads/month">
-</a>
-</p>
 
-<h2 align="center">Open-source framework for data-centric NLP</h2>
-<p align="center">Data Labeling, curation, and Inference Store</p>
-<p align="center">Designed for MLOps & Feedback Loops</p>
+> This Docker image corresponds to the **Argilla Hugging Face Spaces deployment** and **can only be used to deploy Argilla inside the Hugging Face Hub**. For other type of deployments check the Argilla docs.
```

Comment:
If this is true should we rename the name of the Dockerfile to something like `hf-spaces` or similar?